Week3/homework/step2-2.js

```diff
@@ -2,19 +2,23 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  console.log(numbers);
```

Comment:
This console.log (used for debugging purposes) should be removed when you submit your code in a pull request.